test/run-locally.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+echo "starting vms"
```

Comment:
please add a helper/ echo message in case the argument numbers is not the one the script would expect

Reply:
I think I will just pass everything to ginkgo directly.

Reply:
Can't we do a ginkgo --dry-run before provisioning the VMs? It would be annoying if we would have to provision the VMs and then find out that ginkgo fails because of some random thing.

Reply:
Great idea!

Reply:
This would just check that that the ginkgo code compiles, right? (i.e. the test code will run). It won't do the before* stuff (and set up the VMs).